Lib/test/test_types.py

```diff
@@ -597,6 +597,12 @@ def test_slot_wrapper_types(self):
         self.assertIsInstance(object.__lt__, types.WrapperDescriptorType)
         self.assertIsInstance(int.__lt__, types.WrapperDescriptorType)
 
+    def test_dunder_get_signature(self):
+        sig = inspect.signature(object.__init__.__get__)
+        self.assertEqual(list(sig.parameters), ["instance", "owner"])
+        # gh-93021: Second parameter is optional
+        self.assertIs(sig.parameters["owner"].default, None)
```

Comment:
Nit: we have `self.assertIsNone`

Reply:
Oops, good catch. I feel like it's not worth a followup PR, but I'll keep it in mind for next time.